src/components/course/CoursePage.jsx

```diff
@@ -26,7 +26,7 @@ const CoursePage = () => {
   ]);
 
   // If there isn't an active course run we don't show the course at all
-  if (!courseMetadata.activeCourseRun) {
+  if (!courseMetadata?.activeCourseRun) {
```

Comment:
Resolves a separate bug if you load course page with bogus course key and `courseMetadata` is expected to be `null` or `undefined`.